rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -605,19 +629,33 @@
     }
 
     $(function($) { // onLoad
+      // there can be only one
+      if (document.querySelector(tooltipContainerSel) === null) {
```

Comment:
I don't think the `tooltipContainerSel` variable nor this conditional are necessary now that you created the `$tooltipContainer` variable

Reply:
It's necessary for the case where there are two rapid response asides on the page

Reply:
Actually given that we will need to make sure `$tooltipContainer` points to the existing one if we didn't prepend it here

Reply:
Ah, of course. Good catch. I guess it's best to create the variable without a value, then set it to the new element or the existing element here on page lod